src/ImageSharp/Formats/Png/DeflateCompressionLevel.cs

```diff
@@ -6,7 +6,7 @@ namespace SixLabors.ImageSharp.Formats.Png
     /// <summary>
     /// Provides enumeration of available PNG compression levels.
     /// </summary>
-    public enum PngCompressionLevel
+    public enum DeflateCompressionLevel
```

Comment:
While reasonable, this will be a hard breaking change, introducing an experimental codec, doesn't justify it for the rest of the users IMO. Can we try a softer way? 
- Duplicating `DeflateCompressionLevel` as `PngCompressionLevel` while marking the latter `[EditorBrowsable(false)]`
- Duplicating a `PngCompressionLevel` variant of `PngEncoder.CompressionLevel` and marking it `[EditorBrowsable(false)]`

We [may or may not want](https://github.com/SixLabors/ImageSharp/discussions/1454) to obsolete the old API-s, no strong opinion.

Reply:
yes you are right, i will change that back

Reply:
Changing back is fine for now. Later we may still want to introduce the "soft" ranaming, since `PngCompressionLevel` is weird when using with TIFF :)

Reply:
I have duplicated `PngCompressionLevel` to `DeflateCompressionLevel` and marked `PngCompressionLevel` as `[EditorBrowsable(EditorBrowsableState.Never)]`.

I was unsure if that's what you suggested. Is that 7357ae38 ok?

Reply:
Also wanted to somehow duplicate the `PngEncoder.CompressionLevel` but did not think it through. We can't have two properties of the same name, not sure how to proceed now.

Reply:
If noone can figure it out, lets revert to the old name for the first release maybe?

Reply:
`PngEncoder.CompressionLevel` was already changed back with a09cefe080e8f86ae60a12a16adb8f509c17b97d